go/libkb/output_windows.go

```diff
@@ -92,6 +92,12 @@ type ColorWriter struct {
 	mutex *sync.Mutex
 }
 
+// Fd returns the underlying file descriptor. This is mainly to
```

Comment:
The whole terminal detection thing? Looks to have Windows support, we'll see. If we want to gate the colors just on this instead of some other criteria, maybe we could take out some of the ColorWriter plumbing. LGTM.